README.md

```diff
@@ -12,56 +12,70 @@ Since the very beginning, the Ada language has been formally defined through a p
+If there is an issue that you have not found discussed, please feel welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
 
-It is encouraged that issues first be put to discussion as a Github Issue in this repository, to get feedback from the community, and possibly to identify any early issues, before forming a Study Group. 
+If you find an issue is not gaining enough traction, or has received no comments after a month, considering "pinging" the issue generate notifications to anyone following the issue or the repository.
 
-Anyone who wishes to form a Language Study Group to work towards a formal AI should submit a request through the official ARG [web-form](https://arg.adaic.org/community-input).
+For issues that gain significant traction, it may be encouraged to propose the creation of a Language Study Group to develop the idea further, and ultimately into an AI. See the Language Study Groups section below.
 
-When a new Language Study Group is approved by the ARG, a Google Docs shared folder will be created. This folder should remain publicly readable, and should contain all working documents used by the Study Group to carry-out its work.
+## Direct Induction of Issues by the ARG
 
-A list of active Study Groups can be found [here](https://arg.adaic.org/community-input).
+Members of the ARG monitor and comment on issues posted to this Github repository. ARG members strive to comment on all issues, but sometimes issues get missed. 
 
+The ARG may, from time-to-time, decide to directly induct a community issue into an AI without requiring the creation of a Language Study Group. In particular, the ARG may be interested in issues that identify obvious errors/flaws/bugs in the existing Ada standard. At other times, the ARG or its members may identify an issue that has particular merit but that has not attracted sufficient discussion or interest from the community to lead to a Language Study Group proposal.
 
----
-# How can I start?
+If the ARG later finds that such an AI needs significant work, the ARG might vote that the AI requires the creation of a Language Study Group to proceed. At that point the original owner of the original issue is encouraged to lead that effort.
 
-The best way to get started is to browse open issues to get a sense of how issues are structured, and to see if you find any existing issues that interest you. Consider speaking with supporters within that discussion about proposing a Language Study Group to take the issue further.
+## Language Study Groups
 
-If there is an issue that you have not found discussed, please be welcome to open a new issue. Try to give it a concise title, a succinct description, and an appropriate tag.
+Language Study Groups are created to transparently capture the work towards the generation and ongoing development of an AI to eventually be deliberated and voted on by the ARG.
```

Comment:
"... are created to ..." ==> "... are intended to ..."

"... towards the generation and ..." ==> "... towards the creation and ..."

Reply:
Fixed